src/applications/disability-benefits/526EZ/containers/ConfirmationPage.jsx

```diff
@@ -61,53 +60,49 @@ class ConfirmationPage extends React.Component {
     return (
       <div>
         <h3 className="confirmation-page-title">Your claim has been submitted.</h3>
-        <p>We usually process claims within <strong>99 days</strong>.</p>
-        <p>We may contact you if we have questions or need more information.
-        You can print this page for your records.</p>
-        { selected4142 && privateRecordReleaseContent}
-        <h4 className="confirmation-guidance-heading">
-          What happens after I apply?
-        </h4>
-        <p className="confirmation-guidance-message">
-          You don’t need to do anything unless we send you a letter asking for
-          more information. You can check the status of your claim online. The
-          time frame you see may vary based on how complex your claim is.
+        <p>
+          We process applications in the order we receive them.
+          We may contact you if we have questions or need more information.
+          You can print this page for your records.
         </p>
-        <a href="/disability-benefits/track-claims">
-          Check the status of your disability claim.
-        </a>
-        <br/>
-        <a href="/disability-benefits/after-you-apply/">
-          Learn more about what happens after you file a disability claim.
-        </a>
+        { selected4142 && privateRecordReleaseContent}
+
         <div className="inset">
-          <h4>Disability Compensation Claim for Increase <span className="additional">(Form 21-526EZ)</span></h4>
+          <h4>Disability Compensation Claim <span className="additional">(Form 21-526EZ)</span></h4>
           <span>
             For {first} {middle} {last} {suffix}
           </span>
           <ul className="claim-list">
-            <strong>Conditions claimed</strong>
+            <strong>Conditions claimed for increase</strong>
```

Comment:
I think this probably shouldn't show unless there's been a successful submission. If there's a submission error, this heading still shows up, but the disabilities list under it will be empty and there'll be an error message pushed into `{this.props.submissionMessage}`, which would look kind of strange.

Reply:
Do you mean it shouldn't show unless there's a successful submission to EVSS or the api? If we get to the confirmation page, we've successfully submitted to the api, we should have the list of conditions.

Have you run into this problem before?